test/benchmark_utils/test_benchmark_utils.py

```diff
@@ -436,7 +436,7 @@ class MockCudaTimer(benchmark_utils.Timer):
               20 measurements, 1000 runs per measurement, 16 threads"""
         )
 
-    @slowTest
+    # Just to make sure this test runs in PR ci will be added back before merging
```

Comment:
can't you put this on the ci-all branch?

Reply:
Will do that for the valgrind PR